src/librustdoc/lib.rs

```diff
@@ -65,19 +65,8 @@ mod doctree;
 #[macro_use]
 mod error;
 mod fold;
-mod formats;
-pub mod html {
-    crate mod escape;
-    crate mod format;
-    crate mod highlight;
-    crate mod item_type;
-    crate mod layout;
-    pub mod markdown;
-    crate mod render;
-    crate mod sources;
-    crate mod static_files;
-    crate mod toc;
-}
+crate mod formats;
+crate mod html;
```

Comment:
Is librustdoc meant to be dependended on from other repos? Changing visibility of a previously exported module is quite drastic so just want to make.

Reply:
AFAIK rustdoc is only used as a library from `src/tools/rustdoc`. I would be very surprised if anyone else was depending on it.

Reply:
librustdoc is not part of the sysroot, so you will need to compile it yourself if you want to use it.

Reply:
it turns out making it `crate` breaks a doc compiletest in `html/markdown.rs` so i had to revert it to `pub` anyway